modules/authorization/policy-assignments/resource-group/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Policy Assignment on Resource Group level",
+  "summary": "With this module you can perform policy assignments on a resource group level",
```

Comment:
Should the README.md be updated to "This module deploys Policy Assignments on a Resource Group level." and then this file updated accordingly?